### PR TITLE
✨ Show/hide data

### PR DESCRIPTION
**Pull request by LouisaStrunk02**
Changes:

1. add ids
2. add function "visibility"

### CODE REVIEW

File: table.html [hunk 1]
```diff
@@ -11,34 +11,34 @@
     <table class="table">
       <caption class="table__caption">HTML-Table</caption>
       <tr class="table__head">
-        <th class="table-head__cell">Name</th>
-        <th class="table-head__cell">Age</th>
-        <th class="table-head__cell">Phone</th>
-        <th class="table-head__cell">Email</th>
+        <th id="name" onclick="visibility(this.id)" class="table-head__cell">Name</th>
```
Comment (DevBastiS) [inline table.html:14]: ids have to be unique.

**Review by DevBastiS — CHANGES_REQUESTED** [review]

File: table.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function visibility(columnId) {
```
Comment (stefanluth) [inline table.js:1]: 2 spaces indents

File: table.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 function visibility(columnId) {
```
Comment (stefanluth) [inline table.js:1]: Function names should include a verb to show what the function does :) E.g.

`changeVisibility` or `toggleVisibility`

File: table.html [hunk 4]
```diff
@@ -17,28 +17,28 @@
         <th id="mail" onclick="visibility(this.id)" class="table-head__cell">Email</th>
       </tr>
       <tr class="table__body">
-        <td id="name" class="table-body__cell">Hans</td>
-        <td id="age" class="table-body__cell">47</td>
-        <td id="phone" class="table-body__cell">0123456789</td>
-        <td id="mail" class="table-body__cell">Hans@Email</td>
+        <td name="name" class="table-body__cell">Hans</td>
```
Comment (stefanluth) [inline table.html:20]: According to w3schools, the `td`-tag does not support the `name`-attribute; It only supports [these](https://www.w3schools.com/tags/tag_td.asp) and the `[global attributes](https://www.w3schools.com/tags/ref_standardattributes.asp)`.

**Review by stefanluth — CHANGES_REQUESTED** [review]

File: table.html (hunk 4 above)
Comment (stefanluth) [reply table.html:20]: Depending on what the intended purpose of the "name"-tag was, it could be enough to place it in the class name instead.

**Review by DevBastiS — APPROVED** [review]
What @stefanluth said

**Review by stefanluth — APPROVED** [review]
lgtm :)

**Pull request merged**